src/data/entities/Profile.ts

```diff
@@ -27,7 +27,7 @@ export class Profile extends BaseEntry {
   phone: string | null = null
 
   @Column({ type: 'bigint' })
-  placeId: number | null = null
+  place_id: number | null = null // eslint-disable-line
```

Comment:
why? this is a consistency problem. do we have to do it this way?
[column decorator allows using names](https://github.com/typeorm/typeorm/blob/master/docs/decorator-reference.md#column)

Reply:
The purpose is to load relation id without joining relation: https://github.com/typeorm/typeorm/blob/master/docs/relations-faq.md#how-to-use-relation-id-without-joining-relation
In the db relation id field name is created as `place_id`, therefore I changed the field name. I agree that it is not nice. I think it is the naming convention of mysql. Hibernate has an option to define relation's id field name, if typeorm has such an feature we can update it. https://stackoverflow.com/a/3964086/1027846

Reply:
I made an update on gql type definition, and initial version is fine now. I rollbacked the change.